package.json

```diff
@@ -1,28 +1,39 @@
 {
   "private": true,
   "scripts": {
+    "build:app-shared": "yarn workspace @openvaa/app-shared build && true",
```

Comment:
Adds scripts for building shared dependencies which are needed for linting or generating the translation key types.